sql/plan/show_create_table.go

```diff
@@ -255,7 +266,13 @@ func (i *showCreateTablesIter) produceCreateTableStatement(ctx *sql.Context, tab
 
 		// TODO: The columns that are rendered in defaults should be backticked
 		if col.Default != nil {
-			stmt = fmt.Sprintf("%s DEFAULT %s", stmt, col.Default.String())
+			def := col.Default.String()
```

Comment:
Do we know how the double-quotes get stored? I noticed in one of the test cases that the input statement used single quotes, but was output wrapped in double-quotes.

Reply:
I think go stores string value like `val = "example"`, and when this gets used with String() method of literal expressions, we use `%q` for `fmt.Sprintf` method for string type, which double quotes the double quotes, which makes it `"\"example\""`.

Reply:
Is there something we should do at this level to prevent double quotes from being stored? I haven't dug through this code yet to understand it deeply, but I'm curious what you think. Do you see a good reason we're storing the double quotes? 

Overall, I think this fix is okay, but just wanted to understand the rest of the code. I'm specifically wondering if there are other problems from storing the double quotes that we could fix if we looked a little deeper here.